helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -419,6 +424,10 @@ clusterReceiver:
   terminationGracePeriodSeconds: 600
   priorityClassName: ""
 
+  # Container configurations
+  container:
```

Comment:
No need to add container-level securityContext. All the container by default inherits pod-level securityContext.